scanning/audit-rules-syscalls-grouping/test.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python3
+
+import re
+
+from lib import util, results, versions
+
+syscalls_groups = [
+    ['setxattr', 'lsetxattr', 'fsetxattr', 'removexattr', 'lremovexattr', 'fremovexattr'],
+    ['init_module', 'delete_module', 'query_module', 'finit_module'],
+    ['open', 'creat', 'truncate', 'ftruncate', 'openat', 'open_by_handle_at'],
+    ['lchown', 'fchown', 'chown', 'fchownat'],
+    ['adjtimex', 'settimeofday'],
+    ['clock_settime'],
+    ['sethostname', 'setdomainname'],
+]
+# fchmodat2 is only available since RHEL 10+ kernels,
+# CaC/content rule for renameat2 is only on RHEL 10+
+if versions.rhel >= 10:
+    syscalls_groups.append(['chmod', 'fchmod', 'fchmodat', 'fchmodat2'])
+    syscalls_groups.append(['unlink', 'unlinkat', 'rename', 'renameat', 'renameat2', 'rmdir'])
+else:
+    syscalls_groups.append(['chmod', 'fchmod', 'fchmodat'])
+    syscalls_groups.append(['unlink', 'unlinkat', 'rename', 'renameat', 'rmdir'])
+
+
+def syscalls_pretty_print(syscalls):
+    return ','.join(syscalls)
+
+
+def unselect_all_rules_except_audit(orig_ds, new_ds):
+    rule_line = re.compile(r'<[^>]*Rule[^>]*\bid="xccdf_org.ssgproject.content_rule_')
+    audit_rules = re.compile(
+        r'<.*Rule.*id="xccdf_org.ssgproject.content_rule_audit_rules.*'
+        r'(_unsuccessful_file_modification|_dac_modification|'
+        r'_file_deletion_events|_kernel_module_loading|'
+        r'_networkconfig_modification|_time_adjtimex|'
+        r'_time_clock_settime|_time_settimeofday|_time_stime)',
+    )
+
+    with open(orig_ds) as orig_ds_f:
+        with open(new_ds, 'w') as new_ds_f:
```

Comment:
This could be also written as:
```suggestion
    with open(orig_ds) as orig_ds_f, open(new_ds, 'w') as new_ds_f:
```